build/db-schemas/000004_drop_cells_table.up.sql

```diff
@@ -0,0 +1,3 @@
+DROP TABLE IF EXISTS cells_identification_service_areas;
+DROP TABLE IF EXISTS cells_subscriptions;
+UPDATE schema_versions set schema_version = 'v2.0.0' WHERE onerow_enforcer = TRUE
```

Comment:
We have made changes that rely on versioning, this tool allows for backwards compatibility and it will safely migrate all existing data to the "v2.0.0", or down to "v1.0.0" if necessary. This means that for those who already ran the old binary can freely upgrade without much work, the new deployment jsonnet configs that I'm working on would make it easier as well.

Reply:
It seems weird that we're going to have two database schemas that are both considered "v1.0.0" and two database schemas that are both considered "v2.0.0" (each one with a schema_versions table + schema_migrations table, and each one without both), but actually it does look like an original-v2.0.0 configuration can be migrated to to a migrations-v2.0.0 configuration by applying all the up steps from the beginning (what will happen when the migration version isn't detected correctly, IIUC), and likewise for original-v1.0.0.  I think it would help a lot to document the intended usage and expected behavior of this tool under normal circumstances and under anticipated legacy cases like these in a README, but perhaps that is best suited for a separate PR to limit PR scope.  And since I was wrong about backwards compatibility, I'll resolve this comment.